Example/Pods/UberSignals/UberSignals/UBSignal.h

```diff
@@ -195,6 +195,7 @@ CreateSignalInterface(UBMutableDictionarySignal, NSMutableDictionary *mutableDic
 - (instancetype)initWithProtocol:(Protocol *)protocol NS_DESIGNATED_INITIALIZER;
 
 - (instancetype)init NS_UNAVAILABLE;
++ (instancetype)new NS_UNAVAILABLE;
```

Comment:
Perfect, thanks for submitting this!  Just sign the CLA and then we just need one more approval.